src/background.js

```diff
@@ -0,0 +1,20 @@
+function isGithub(tab) {
+    return tab.url && tab.url.startsWith('https://github.com') != -1 && tab.url.indexOf('/blob/') != -1;
+};
+
+function runShowMeTheCode(tabId, tab) {
+    if (isGithub(tab)) {
+        chrome.tabs.insertCSS(tabId, {file: 'showMeTheCode.css'});
+    } else {
+        chrome.tabs.insertCSS(tabId, {file: 'disableShowMeTheCode.css'});
+    }
+};
+
+chrome.tabs.onUpdated.addListener(function (tabId, info, tab) {
```

Comment:
It may be possible to attach listener to more appropriate event but I didn't find it